git-tar/function/handler.go

```diff
@@ -219,3 +229,22 @@ func reportStatus(status *sdk.Status) {
 		log.Printf("failed to report status, error: %s", reportErr.Error())
 	}
 }
+
+func formatTemplatePath(clonePath string) string {
+	folders := strings.Split(clonePath, "/")
+	return fmt.Sprintf("/%s", folders[1])
+}
+
+func copyTemplates(templatePath, clonePath string) error {
+	copyCmd := exec.Command("cp", "-r", "templates", clonePath)
```

Comment:
Please do not execute `cp`. See the approach we take in the faas-cli. If in doubt you can ask @rgee0 and @LucasRoesler about this since they have worked on.

Reply:
Their approach was my formal one but we discussed and Rosti told that `cp` uses some linux optimization.I will remake it how its done with `faas-cli`.